docs/sw-update-conf.json

```diff
@@ -80,7 +102,13 @@
           "pip": "https://github.com/mrbeam/netconnectd_mrbeam/archive/{target_version}.zip",
           "global_pip_command": true,
           "package_name": "netconnectd",
-          "branch": "mrbeam2-stable"
+          "branch": "mrbeam2-stable",
+          "beamos_date": {
+            "2021-06-11": {
+              "pip_command": "sudo /usr/local/netconnectd/venv/bin/pip",
+              "branch": "master"
```

Comment:
what do you mean? This will override the mrbeam2-stable that is used for the legacy image. But you are right I should check if this be handeld correct for the alpha, beta, develop branch for the buster image